flink-metrics/flink-metrics-influxdb/src/main/java/org/apache/flink/metrics/influxdb/InfluxdbReporter.java

```diff
@@ -77,10 +81,15 @@ public void open(MetricConfig config) {
 
 		this.database = database;
 		this.retentionPolicy = getString(config, RETENTION_POLICY);
+
+		int connectTimeout = getInteger(config, CONNECT_TIMEOUT);
+		int writeTimeout = getInteger(config, WRITE_TIMEOUT);
+		OkHttpClient.Builder client = new OkHttpClient.Builder().connectTimeout(connectTimeout, TimeUnit.MICROSECONDS).writeTimeout(writeTimeout, TimeUnit.MICROSECONDS);
```

Comment:
Time unit should be `TimeUnit.MILLISECONDS`.

Also, the line is a little bit too long. I suggest to reformat it like the following (also inlining the config keys lookups):
```java
		OkHttpClient.Builder client = new OkHttpClient.Builder()
			.connectTimeout(getInteger(config, CONNECT_TIMEOUT), TimeUnit.MILLISECONDS)
			.writeTimeout(getInteger(config, WRITE_TIMEOUT), TimeUnit.MILLISECONDS);
```